### PR TITLE
Add missing descriptions for nodes in xod-dev/w5500, xod/net and xod/stream

### PR DESCRIPTION
**Pull request by evgenykochetkov**

### CODE REVIEW

File: workspace/__lib__/xod/stream/accumulate-string/patch.xodp [hunk 1]
```diff
@@ -31,6 +33,7 @@
       "type": "xod/patch-nodes/not-implemented-in-xod"
```
Comment (nkrkv) [inline workspace/__lib__/xod/stream/accumulate-string/patch.xodp]: A comment for `CAP`: “Capacity of the string buffer. Defines the maximum length. Must be a constant…”

File: workspace/__lib__/xod/stream/parse-integer/patch.xodp [hunk 2]
```diff
@@ -1,6 +1,8 @@
 {
+  "description": "Reads an integer number from a stream of characters. Stops when a non-numeric character was encountered",
```
Comment (nkrkv) [inline workspace/__lib__/xod/stream/parse-integer/patch.xodp]: Prefer present tense

File: workspace/__lib__/xod/stream/pass-from-sequence/patch.xodp [hunk 3]
```diff
@@ -152,6 +155,7 @@
       "boundLiterals": {
         "Hy9qN5AZm": "\"\""
       },
+      "description": "Searched sequence",
```
Comment (nkrkv) [inline workspace/__lib__/xod/stream/pass-from-sequence/patch.xodp]: This description should be on the `SEQ` terminal, not on `pulse-on-sequence`

File: workspace/__lib__/xod/net/format-http-request/patch.xodp [hunk 4]
```diff
@@ -246,6 +247,7 @@
       "boundLiterals": {
         "__out__": "\"\""
       },
+      "description": "The domain name of the server and the TCP port number on which the server is listening(like xod.io:80)",
```
[the remark this answers is not in the document]
Comment (evgenykochetkov) [reply workspace/__lib__/xod/net/format-http-request/patch.xodp]: [Yep.](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Host)
```
Syntax
Host: <host>:<port>
```

**Comment by evgenykochetkov** [comment]
@nkrkv done, check again please

File: workspace/__lib__/xod-dev/w5500/close/patch.xodp [hunk 5]
```diff
@@ -1,6 +1,8 @@
 {
+  "description": "Closes TCP connection",
```
Comment (nkrkv) [inline workspace/__lib__/xod-dev/w5500/close/patch.xodp]: Closes an open TCP connection

File: workspace/__lib__/xod-dev/w5500/close/patch.xodp [hunk 6]
```diff
@@ -1,6 +1,8 @@
...
   "nodes": [
     {
+      "description": "Pulses when connection is closed",
```
Comment (nkrkv) [inline workspace/__lib__/xod-dev/w5500/close/patch.xodp]: the connection

File: workspace/__lib__/xod-dev/w5500/close/patch.xodp [hunk 7]
```diff
@@ -10,15 +12,17 @@
       "type": "xod/patch-nodes/output-pulse"
     },
     {
+      "description": "Close connection",
```
Comment (nkrkv) [inline workspace/__lib__/xod-dev/w5500/close/patch.xodp]: the connection

File: workspace/__lib__/xod-dev/w5500/close/patch.xodp [hunk 8]
```diff
@@ -10,15 +12,17 @@
...
-      "label": "STOP",
+      "label": "CLS",
       "position": {
         "x": 68,
         "y": 0
       },
       "type": "xod/patch-nodes/input-pulse"
     },
     {
+      "description": "Socket",
```
Comment (nkrkv) [inline workspace/__lib__/xod-dev/w5500/close/patch.xodp]: The socket

File: workspace/__lib__/xod-dev/w5500/connect-manual/patch.xodp [hunk 9]
```diff
@@ -1,6 +1,8 @@
 {
+  "description": "Establish internet connection with manual configuration",
```
Comment (nkrkv) [inline workspace/__lib__/xod-dev/w5500/connect-manual/patch.xodp]: an internet connection configured manually

File: workspace/__lib__/xod-dev/w5500/connect-manual/patch.xodp [hunk 10]
```diff
@@ -1,6 +1,8 @@
...
   "nodes": [
     {
+      "description": "Internet connection",
```
Comment (nkrkv) [inline workspace/__lib__/xod-dev/w5500/connect-manual/patch.xodp]: The internet connection

File: workspace/__lib__/xod-dev/w5500/connect-manual/patch.xodp [hunk 11]
```diff
@@ -18,6 +20,7 @@
       "type": "xod/patch-nodes/not-implemented-in-xod"
     },
     {
+      "description": "Pulses on successful connection",
```
Comment (nkrkv) [inline workspace/__lib__/xod-dev/w5500/connect-manual/patch.xodp]: a successful connection

File: workspace/__lib__/xod-dev/w5500/connect-manual/patch.xodp [hunk 12]
```diff
@@ -27,6 +30,7 @@
       "type": "xod/patch-nodes/output-pulse"
     },
     {
+      "description": "IP address of the gateway. By default is the same as IP but with last octet being '1'",
```
Comment (nkrkv) [inline workspace/__lib__/xod-dev/w5500/connect-manual/patch.xodp]: An IP address of the gateway. Defaults to `IP` with the last octet set to `1`.

File: workspace/__lib__/xod-dev/w5500/connect-manual/patch.xodp [hunk 13]
```diff
@@ -39,15 +43,17 @@
       "boundLiterals": {
         "__out__": "On Boot"
       },
+      "description": "Establish connection",
```
Comment (nkrkv) [inline workspace/__lib__/xod-dev/w5500/connect-manual/patch.xodp]: Pulse to initiate a connection

File: workspace/__lib__/xod-dev/w5500/connect-manual/patch.xodp [hunk 14]
```diff
@@ -39,15 +43,17 @@
...
-      "label": "INIT",
+      "label": "CONN",
       "position": {
         "x": 374,
         "y": 0
       },
       "type": "xod/patch-nodes/input-pulse"
     },
     {
+      "description": "Subnet mask. 255.255.255.0 by default",
```
Comment (nkrkv) [inline workspace/__lib__/xod-dev/w5500/connect-manual/patch.xodp]: A subnet mask…

File: workspace/__lib__/xod-dev/w5500/connect-manual/patch.xodp [hunk 15]
```diff
@@ -57,6 +63,7 @@
       "type": "xod/net/input-ip-address"
     },
     {
+      "description": "IP address of the DNS server. By default is the same as IP but with last octet being '1'",
```
Comment (nkrkv) [inline workspace/__lib__/xod-dev/w5500/connect-manual/patch.xodp]: An IP address… the last octet…

File: workspace/__lib__/xod-dev/w5500/connect-manual/patch.xodp [hunk 16]
```diff
@@ -66,6 +73,7 @@
       "type": "xod/net/input-ip-address"
     },
     {
+      "description": "Pulses if connection failed",
```
Comment (nkrkv) [inline workspace/__lib__/xod-dev/w5500/connect-manual/patch.xodp]: the connection

File: workspace/__lib__/xod-dev/w5500/connect-manual/patch.xodp [hunk 17]
```diff
@@ -84,6 +93,7 @@
       "type": "xod/net/input-ip-address"
     },
     {
+      "description": "Device configuration",
```
Comment (nkrkv) [inline workspace/__lib__/xod-dev/w5500/connect-manual/patch.xodp]: A W5500 internet provider device

**Comment by evgenykochetkov** [comment]
Done 🙌🏻
download-1

@nkrkv @brusherru please check again

File: workspace/__lib__/xod-dev/w5500/ethernet-shield-device/patch.xodp [hunk 18]
```diff
@@ -5,7 +5,7 @@
       "boundLiterals": {
         "__out__": "D10"
       },
-      "description": "Chip select port",
+      "description": "The chip select port",
```
Comment (nkrkv) [inline workspace/__lib__/xod-dev/w5500/ethernet-shield-device/patch.xodp]: A chip select port

File: workspace/__lib__/xod-dev/w5500/ethernet-shield-device/patch.xodp [hunk 19]
```diff
@@ -33,7 +33,7 @@
       "type": "xod/patch-nodes/not-implemented-in-xod"
     },
     {
-      "description": "MAC address of the shield",
+      "description": "The MAC address",
```
Comment (nkrkv) [inline workspace/__lib__/xod-dev/w5500/ethernet-shield-device/patch.xodp]: A MAC address. Should be unique for a device. Often printed on the bottom or a package.

File: workspace/__lib__/xod-dev/w5500/http-request(inet)/patch.xodp [hunk 20]
```diff
@@ -290,7 +290,7 @@
       "boundLiterals": {
         "__out__": "80"
       },
-      "description": "TCP port number on which the server is listening",
+      "description": "The TCP port number on which the server is listening",
```
Comment (nkrkv) [inline workspace/__lib__/xod-dev/w5500/http-request(inet)/patch.xodp]: A port number on which the server is listening. In most configurations, HTTP web servers listen at port 80.

File: workspace/__lib__/xod-dev/w5500/http-request(inet)/patch.xodp [hunk 21]
```diff
@@ -341,7 +341,7 @@
       "boundLiterals": {
         "__out__": "\"\""
       },
-      "description": "Request body(optional)",
+      "description": "The request body(optional)",
```
Comment (nkrkv) [inline workspace/__lib__/xod-dev/w5500/http-request(inet)/patch.xodp]: Space before `(`

File: workspace/__lib__/xod-dev/w5500/http-request(inet)/patch.xodp [hunk 22]
```diff
@@ -364,7 +364,7 @@
       "boundLiterals": {
         "__out__": "\"/\""
       },
-      "description": "Path to requested resource",
+      "description": "The path to the requested resource",
```
Comment (nkrkv) [inline workspace/__lib__/xod-dev/w5500/http-request(inet)/patch.xodp]: A path

File: workspace/__lib__/xod-dev/w5500/http-request(inet)/patch.xodp [hunk 23]
```diff
@@ -428,7 +428,7 @@
       "type": "xod/patch-nodes/output-pulse"
     },
     {
-      "description": "Internet connection",
+      "description": "The internet connection",
```
Comment (nkrkv) [inline workspace/__lib__/xod-dev/w5500/http-request(inet)/patch.xodp]: An

File: workspace/__lib__/xod-dev/w5500/http-request(inet)/patch.xodp [hunk 24]
```diff
@@ -451,7 +451,7 @@
       "boundLiterals": {
         "__out__": "\"GET\""
       },
-      "description": "HTTP method (e.g., GET, POST, PUT, or DELETE)",
+      "description": "The HTTP method (e.g., GET, POST, PUT, or DELETE)",
```
Comment (nkrkv) [inline workspace/__lib__/xod-dev/w5500/http-request(inet)/patch.xodp]: An

File: workspace/__lib__/xod-dev/w5500/is-open/patch.xodp [hunk 25]
```diff
@@ -22,7 +22,7 @@
       "type": "xod/patch-nodes/output-pulse"
     },
     {
-      "description": "Socket",
+      "description": "The socket",
```
Comment (nkrkv) [inline workspace/__lib__/xod-dev/w5500/is-open/patch.xodp]: A

File: workspace/__lib__/xod-dev/w5500/lan-ip/patch.xodp [hunk 26]
```diff
@@ -1,5 +1,5 @@
 {
-  "description": "Gets local IP address",
+  "description": "Gets the local IP address",
```
Comment (nkrkv) [inline workspace/__lib__/xod-dev/w5500/lan-ip/patch.xodp]: a

File: workspace/__lib__/xod-dev/w5500/open-tcp/patch.xodp [hunk 27]
```diff
@@ -12,7 +12,7 @@
       "type": "xod/patch-nodes/output-pulse"
     },
     {
-      "description": "Socket",
+      "description": "The socket",
```
Comment (nkrkv) [inline workspace/__lib__/xod-dev/w5500/open-tcp/patch.xodp]: The socket connected

File: workspace/__lib__/xod-dev/w5500/open-tcp/patch.xodp [hunk 28]
```diff
@@ -32,7 +32,7 @@
       "type": "xod/patch-nodes/input-pulse"
     },
     {
-      "description": "Internet connection",
+      "description": "The internet connection",
```
Comment (nkrkv) [inline workspace/__lib__/xod-dev/w5500/open-tcp/patch.xodp]: An

File: workspace/__lib__/xod-dev/w5500/open-tcp/patch.xodp [hunk 29]
```diff
@@ -76,7 +76,7 @@
       "boundLiterals": {
         "__out__": "80"
       },
-      "description": "Port number",
+      "description": "The port number",
```
Comment (nkrkv) [inline workspace/__lib__/xod-dev/w5500/open-tcp/patch.xodp]: A destination server port number

File: workspace/__lib__/xod-dev/w5500/read-byte/patch.xodp [hunk 30]
```diff
@@ -1,8 +1,8 @@
 {
-  "description": "Read a single byte of response from opened connection",
+  "description": "Read a single byte of the response from the opened connection",
```
Comment (nkrkv) [inline workspace/__lib__/xod-dev/w5500/read-byte/patch.xodp]: an opened connection

File: workspace/__lib__/xod-dev/w5500/read-byte/patch.xodp [hunk 31]
```diff
@@ -12,7 +12,7 @@
       "type": "xod/patch-nodes/input-pulse"
     },
     {
-      "description": "Socket",
+      "description": "The socket",
```
Comment (nkrkv) [inline workspace/__lib__/xod-dev/w5500/read-byte/patch.xodp]: A

File: workspace/__lib__/xod-dev/w5500/receive/patch.xodp [hunk 32]
```diff
@@ -1,5 +1,5 @@
 {
-  "description": "Read response from opened connection as a stream of bytes until it closes",
+  "description": "Read a response from the opened connection as a stream of bytes until it closes",
```
Comment (nkrkv) [inline workspace/__lib__/xod-dev/w5500/receive/patch.xodp]: an opened connection

File: workspace/__lib__/xod-dev/w5500/receive/patch.xodp [hunk 33]
```diff
@@ -264,7 +264,7 @@
       "type": "xod/core/any"
     },
     {
-      "description": "Socket",
+      "description": "The socket",
```
Comment (nkrkv) [inline workspace/__lib__/xod-dev/w5500/receive/patch.xodp]: A

File: workspace/__lib__/xod-dev/w5500/send/patch.xodp [hunk 34]
```diff
@@ -1,8 +1,8 @@
 {
-  "description": "Send message through opened TCP connection",
+  "description": "Send a message through the opened TCP connection",
```
Comment (nkrkv) [inline workspace/__lib__/xod-dev/w5500/send/patch.xodp]: an opened TCP connection

File: workspace/__lib__/xod-dev/w5500/send/patch.xodp [hunk 35]
```diff
@@ -50,7 +50,7 @@
       "type": "xod/patch-nodes/output-pulse"
     },
     {
-      "description": "Socket",
+      "description": "The socket",
```
Comment (nkrkv) [inline workspace/__lib__/xod-dev/w5500/send/patch.xodp]: A

File: workspace/__lib__/xod/net/format-http-request/patch.xodp [hunk 36]
```diff
@@ -299,7 +299,7 @@
       "boundLiterals": {
         "__out__": "\"/\""
       },
-      "description": "Path to requested resource",
+      "description": "A path to the requested resource",
```
Comment (nkrkv) [inline workspace/__lib__/xod/net/format-http-request/patch.xodp]: A path to the requested resource. For example: `/foo/bar?q=13`

File: workspace/__lib__/xod/net/format-http-request/patch.xodp [hunk 37]
```diff
@@ -336,7 +336,7 @@
       "boundLiterals": {
         "__out__": "\"GET\""
       },
-      "description": "HTTP method (e.g., GET, POST, PUT, or DELETE)",
+      "description": "The HTTP method (e.g., GET, POST, PUT, or DELETE)",
```
Comment (nkrkv) [inline workspace/__lib__/xod/net/format-http-request/patch.xodp]: An

File: workspace/__lib__/xod/net/format-http-request/patch.xodp [hunk 38]
```diff
@@ -360,7 +360,7 @@
       "boundLiterals": {
         "__out__": "\"\""
       },
-      "description": "Request body(optional)",
+      "description": "The request body(optional)",
```
Comment (nkrkv) [inline workspace/__lib__/xod/net/format-http-request/patch.xodp]: Space is missing

File: workspace/__lib__/xod/net/format-ip/patch.xodp [hunk 39]
```diff
@@ -1,5 +1,5 @@
 {
-  "description": "Converts IP address to string",
+  "description": "Converts an IP address to string",
```
Comment (nkrkv) [inline workspace/__lib__/xod/net/format-ip/patch.xodp]: a string

File: workspace/__lib__/xod/net/format-ip/patch.xodp [hunk 40]
```diff
@@ -178,7 +178,7 @@
       "type": "xod/core/join"
     },
     {
-      "description": "IP address as string",
+      "description": "The IP address as string",
```
Comment (nkrkv) [inline workspace/__lib__/xod/net/format-ip/patch.xodp]: a string

File: workspace/__lib__/xod/stream/accumulate-string/patch.xodp [hunk 41]
```diff
@@ -1,8 +1,8 @@
 {
-  "description": "Creates a string from a stream of characters by appending each new character to the end of the string until maximum capacity is reached.",
+  "description": "Create a string from a stream of characters by appending each new character to the end of the string until a maximum capacity is reached.",
```
Comment (nkrkv) [inline workspace/__lib__/xod/stream/accumulate-string/patch.xodp]: Creates ... the maximum capacity

File: workspace/__lib__/xod/stream/accumulate-string/patch.xodp [hunk 42]
```diff
@@ -1,8 +1,8 @@
...
   "nodes": [
     {
-      "description": "New character to be pushed at the end of the string",
+      "description": "The new character to be pushed at the end of the string",
```
Comment (nkrkv) [inline workspace/__lib__/xod/stream/accumulate-string/patch.xodp]: A new … pushed to the

File: workspace/__lib__/xod/stream/pass-from-sequence/patch.xodp [hunk 43]
```diff
@@ -1,5 +1,5 @@
 {
-  "description": "Lets characters pass through only after specified sequence is encountered",
+  "description": "Lets the characters pass through only after a specified sequence is encountered",
```
Comment (nkrkv) [inline workspace/__lib__/xod/stream/pass-from-sequence/patch.xodp]: ~the~

File: workspace/__lib__/xod/stream/pass-from-sequence/patch.xodp [hunk 44]
```diff
@@ -134,7 +134,7 @@
       "type": "xod/patch-nodes/input-byte"
     },
     {
-      "description": "Stop passing characters through until next SEQ",
+      "description": "Stop passing the characters through until next SEQ",
```
Comment (nkrkv) [inline workspace/__lib__/xod/stream/pass-from-sequence/patch.xodp]: ~the~

File: workspace/__lib__/xod/stream/pass-if/patch.xodp [hunk 45]
```diff
@@ -1,5 +1,5 @@
 {
-  "description": "Lets values pass through only if COND is true",
+  "description": "Let the values pass through only if COND is true",
```
Comment (nkrkv) [inline workspace/__lib__/xod/stream/pass-if/patch.xodp]: Lets ~the~ values

File: workspace/__lib__/xod/stream/pass-if/patch.xodp [hunk 46]
```diff
@@ -37,7 +37,7 @@
       "type": "xod/patch-nodes/output-pulse"
     },
     {
-      "description": "Decides whether values should be let through",
+      "description": "Decides whether the values should be let through",
```
Comment (nkrkv) [inline workspace/__lib__/xod/stream/pass-if/patch.xodp]: ~the~

File: workspace/__lib__/xod/stream/pass-until/patch.xodp [hunk 47]
```diff
@@ -1,5 +1,5 @@
 {
-  "description": "Lets values pass through until value specified in S is encountered",
+  "description": "Let the values pass through until the value specified in S is encountered",
```
Comment (nkrkv) [inline workspace/__lib__/xod/stream/pass-until/patch.xodp]: ditto

File: workspace/__lib__/xod/stream/pass-until/patch.xodp [hunk 48]
```diff
@@ -162,7 +162,7 @@
       "boundLiterals": {
         "__out__": "On Boot"
       },
-      "description": "Start letting values through until another S",
+      "description": "Start letting the values through until another S",
```
Comment (nkrkv) [inline workspace/__lib__/xod/stream/pass-until/patch.xodp]: Lets values through until another S.

File: workspace/__lib__/xod/stream/pulse-on-sequence/patch.xodp [hunk 49]
```diff
@@ -20,7 +20,7 @@
       "type": "xod/patch-nodes/output-pulse"
     },
     {
-      "description": "Next character in a stream",
+      "description": "The next character in a stream",
```
Comment (nkrkv) [inline workspace/__lib__/xod/stream/pulse-on-sequence/patch.xodp]: the stream

File: workspace/__lib__/xod/stream/skip-until/patch.xodp [hunk 50]
```diff
@@ -1,5 +1,5 @@
 {
-  "description": "Lets values pass through only after value specified in S is encountered",
+  "description": "Let the values pass through only after the value specified in S is encountered",
```
Comment (nkrkv) [inline workspace/__lib__/xod/stream/skip-until/patch.xodp]: Lets values

File: workspace/__lib__/xod/stream/skip-until/patch.xodp [hunk 51]
```diff
@@ -133,7 +133,7 @@
       "type": "xod/core/defer"
     },
     {
-      "description": "Stops letting values through until another S",
+      "description": "Stops letting the values through until another S",
```
Comment (nkrkv) [inline workspace/__lib__/xod/stream/skip-until/patch.xodp]: Do not let values through until another S

File: workspace/__lib__/xod/stream/skip-until/patch.xodp [hunk 52]
```diff
@@ -207,7 +207,7 @@
       "type": "xod/patch-nodes/input-t1"
     },
     {
-      "description": "Value that passed through",
+      "description": "A value that passed through",
```
Comment (nkrkv) [inline workspace/__lib__/xod/stream/skip-until/patch.xodp]: The last value passed through

**Comment by evgenykochetkov** [comment]
Done

**Pull request merged**